xarray/backends/h5netcdf_.py

```diff
@@ -117,13 +118,16 @@ def open(
         lock=None,
         autoclose=False,
         invalid_netcdf=None,
+        phony_dims=None,
     ):
         import h5netcdf
 
         if format not in [None, "NETCDF4"]:
             raise ValueError("invalid format for h5netcdf backend")
 
         kwargs = {"invalid_netcdf": invalid_netcdf}
+        if LooseVersion(h5netcdf.__version__) >= LooseVersion("0.8.0"):
```

Comment:
Let's raise an error otherwise when this condition is not satisfied

Reply:
@dcherian We can do that, but that means users have to use h5netcf >=0.8.0 (which should be OK, I think). Not sure if we should force them. What's the xarray policy for that, BTW? 

Anyway, the whole thing is implemented in h5netcf to be fully backwards compatible. So it should not interfere with users using older versions of h5netcdf.

I'll add the suggested raise first thing tomorrow, if no one stops me.

Reply:
Sorry I wasn't clear enough. What I meant was if a user passes `phony_dims` we should raise if the `h5netcdf` version is not >= 0.8.0

Reply:
@dcherian Ah, OK, that makes absolutely sense. Will do.